install

```diff
@@ -62,20 +57,45 @@ while :; do
 			shift
 		fi
 		;;
+	-P | --prefix)
+		if [ -n "$2" ]; then
+			prefix="$2"
+			shift
+			shift
+		fi
+		;;
 	*) # Default case: If no more options then break out of the loop.
 		break ;;
 	esac
 done
 
+if [ -n "${dest_path}" ] && [ -n "${prefix}" ]; then
+	printf >&2 "Both -p and -P are set. Cannot continue. Exiting.\n"
+	exit 1
+fi
```

Comment:
As discussed in the related issue #184 can we add to this check
if we have dest_path but not prefix, print a warning that the dest_path flags are deprecated

Reply:
Updated in the latest commit